scripts/opportunity-provider/index.ts

```diff
@@ -217,12 +230,9 @@ async function getCallData(
 
 async function submitOpportunity(
   account: PrivateKeyAccount,
-  configPath: string,
-  opportunityPath: string
+  config: Config,
+  opportunity: Opportunity
 ) {
-  const config = readConfig(configPath);
-  const opportunity = readOpportunity(opportunityPath);
-
   const nonce = Math.floor(Math.random() * 2 ** 50);
   const deadline = Math.floor(Date.now() / 1000) + 60 * 60 * 24;
```

Comment:
can we make this `deadline` shorter? i think even a couple minutes should be sufficient and prevents any weird effects on large price movements